DICOMPlugins/DICOMSegmentationPlugin.py

```diff
@@ -205,6 +205,10 @@ def load(self,loadable):
           labelNode.SetAttribute("ColorR", str(rgb[0]))
           labelNode.SetAttribute("ColorG", str(rgb[1]))
           labelNode.SetAttribute("ColorB", str(rgb[2]))
+          if "SegmentAlgorithmType" in segment:
+            labelNode.SetAttribute("SegmentAlgorithmType", segment["SegmentAlgorithmType"])
+          if "SegmentAlgorithmName" in segment:
+            labelNode.SetAttribute("SegmentAlgorithmName", segment["SegmentAlgorithmName"])
```

Comment:
Is there a good reason to not use `DICOM.` prefix here, but then use it later while assigning attributes to the `segmentLabelNode`? Shouldn't it be consistent?

Reply:
`labelNode` is just temporary data, but I'll add the `DICOM.` prefix for readability

Reply:
Great, thanks - I have no other concerns.